tests/ignite/distributed/utils/test_native.py

```diff
@@ -255,21 +255,44 @@ def test_idist_broadcast_gloo(distributed_context_single_node_gloo):
     _test_distrib_broadcast(device)
 
 
+from torch.distributed import GroupMember
+
+
 @pytest.mark.distributed
 @pytest.mark.skipif(not has_native_dist_support, reason="Skip if no native dist support")
 @pytest.mark.skipif(torch.cuda.device_count() < 1, reason="Skip if no GPU")
-def test_idist_barrier_nccl(distributed_context_single_node_nccl):
+@pytest.mark.parametrize(
+    "args,kwargs",
+    [
+        ([], {}),
+        ([GroupMember.WORLD, False], {}),
+        ([GroupMember.WORLD, True], {}),
+        ([GroupMember.NON_GROUP_MEMBER, True], {}),
+        ([GroupMember.NON_GROUP_MEMBER, False], {}),
```

Comment:
@fco-dv what does these checks ?

Reply:
basically wanted to test different input configurations

Reply:
What means `GroupMember.NON_GROUP_MEMBER` ?

Reply:
it's not correct, removing that params set